src/appmixer/microsoft/sharepoint/DownloadFile/DownloadFile.js

```
@@ -1,52 +1,137 @@
+        itemId,
+        itemPath,
+        customFileName
+    } = context.messages.in.content;
+
+    let downloadUrl;
+
+    switch (format) {
+        case 'txt':
+            const metaEndpoint = itemId
+                ? `/drives/${driveId}/items/${itemId}`
+                : `/drives/${driveId}/root:/${itemPath}`;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)